src/mono/browser/runtime/loader/config.ts

```diff
@@ -187,10 +187,13 @@ export function normalizeConfig() {
         config.cachedResourcesPurgeDelay = 10000;
     }
 
+    // ActiveIssue https://github.com/dotnet/runtime/issues/75602
```

Comment:
Unrelated, but shouldn't this come from the tests configuration?

Reply:
You can override it in tests, but at the moment we don't. The issue is to figure out what is the default for customers and then we bump the number for offending tests.